app/assets/scripts/utils/country-centroids.js

```diff
@@ -0,0 +1,247 @@
+  EH: [ -12.730988638417516, 24.181992307542675 ],
+  SA: [ 42.599061141019895, 23.521622162809756 ],
+  SD: [ 30.776868974071107, 14.558226173449514 ],
+  SS: [ 29.904514154535182, 6.918605815777964 ],
+  SN: [ -14.737418070534192, 13.905600113265217 ],
+  SG: [ 103.80524452290484, 1.3471482255238887 ],
+  GS: [ -34.68388168537491, -55.12330371281806 ],
+  SH: [ -10.312877013842014, -24.160988265552493 ],
+  SB: [ 159.58825267307708, -8.86653686052586 ],
+  SL: [ -12.172285488409383, 8.390838918334435 ],
+  SV: [ -88.6669673502936, 13.727297963531626 ],
+  SM: [ 12.43308062875008, 43.931552458125154 ],
+  SO: [ 47.142276150309684, 6.145731601528312 ],
+  PM: [ -56.295459381764616, 46.92474288650013 ],
+  RS: [ 20.710553556498947, 44.21727012375411 ],
+  ST: [ 6.906160186234153, 0.7340788362554724 ],
+  SR: [ -55.97536216504388, 3.986473869809146 ],
+  SK: [ 19.712170290901227, 48.77682673969587 ],
+  SI: [ 14.935461335646327, 46.10792462575013 ],
+  SE: [ 17.556781208837865, 60.8305023027762 ],
+  SZ: [ 31.518478215452973, -26.50009067169799 ],
+  SX: [ -63.07626994684201, 18.043429713022686 ],
+  SC: [ 52.465940066318836, -6.323525617071564 ],
+  SY: [ 37.537073243820366, 35.09020391580417 ],
+  TC: [ -71.85532114706663, 21.734909029822344 ],
+  TD: [ 18.515650773077, 12.180581059082257 ],
+  TG: [ 0.71458170780996, 8.671310447278598 ],
+  TH: [ 100.50721104028825, 12.590776334133894 ],
+  TJ: [ 70.75784497168962, 38.78419140926308 ],
+  TM: [ 57.46212133345149, 39.54581662845143 ],
+  TL: [ 125.59189620071926, -8.851865463039317 ],
+  TO: [ -174.76255839073954, -19.708100579843627 ],
+  TT: [ -61.231980824695995, 10.59901780488252 ],
+  TN: [ 10.015999989692252, 34.7926626652082 ],
+  TR: [ 32.411253727938245, 38.90882391499567 ],
+  TV: [ 178.39537002617675, -7.6887585686468904 ],
+  TW: [ 120.62246559106384, 23.70351151764432 ],
+  TZ: [ 34.89467284375361, -5.4733998155844406 ],
+  UG: [ 32.35512897405116, 0.7800795272119687 ],
+  UA: [ 31.39262238251929, 48.225327040607205 ],
+  UM: [ -59.7244081610525, 15.687877979631738 ],
+  UY: [ -55.94119734047355, -32.6966088942852 ],
+  US: [ -118.2038868585037, 47.910844784195504 ],
+  UZ: [ 65.88606239242623, 41.2226255860016 ],
+  VC: [ -61.25385934591883, 13.022509691919076 ],
+  VE: [ -65.56940462349161, 8.249946252478152 ],
+  VG: [ -64.52193883005705, 18.49092615888585 ],
+  VI: [ -64.81644195358527, 18.12612147087818 ],
+  VN: [ 106.57570438724736, 16.09477989596659 ],
+  VU: [ 167.84305106963384, -16.147551641526285 ],
+  WF: [ -176.90848509328566, -13.662549582428463 ],
+  WS: [ -172.11331132714278, -13.786946593957028 ],
+  YE: [ 47.1086302234567, 14.56738963876404 ],
+  ZA: [ 24.703232827543786, -29.26925840866997 ],
+  ZM: [ 28.489715084919187, -13.149159999209038 ],
+  ZW: [ 29.389317242443635, -18.5589791387905 ]
+};
+
+export function getCentroid (iso) {
+  // Centroid or null island.
+  return centroids[iso.toUpperCase()] || [0, 0];
```

Comment:
We might want to have something here that will prevent mapping things to `[0, 0]`. Maybe returning false would make it easier to catch those?

Reply:
What should be the desired behavior? If we have no centroid don't show the data?
I used null island to present the app from breaking.